aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -435,7 +435,16 @@ module std::features {
     public fun multisig_v2_enhancement_feature_enabled(): bool acquires Features {
         is_enabled(MULTISIG_V2_ENHANCEMENT)
     }
+    
```

Comment:
@alexfilip2 , there is a whitespace which causes the lint error: https://github.com/aptos-labs/aptos-core/actions/runs/8541927236/job/23402440038?pr=12213#step:4:3710